content/app/app-dev-course/modul3/_index.en.md

```diff
@@ -6,7 +6,14 @@ tags: [apps, training, build, deploy, test ]
 weight: 20
 ---
 {{% notice warning %}}
- This module requires that you are part of an [organization](/app/getting-started/create-user/#join-an-organization) with an enabled test environment for Altinn Apps. If this is not the case, move on to the [next module](../modul4/).
+ This module requires that you are part of an [organization](/app/getting-started/create-user/#join-an-organization) with an enabled test environment for Altinn Apps. If this is not the case, move on to the [next module](../modul4/). 
+ <br><br>
+ If you have a personal application registered that you would like to move to an organization, 
+ [you can ask for access to an organization by asking an administrator or the service desk](https://docs.altinn.studio/app/getting-started/create-user/#join-an-organization).
```

Comment:
Let's use relative links so they don't navigate to docs.altinn.studio when running the docs locally
```suggestion
 [you can ask for access to an organization by asking an administrator or the service desk](/app/getting-started/create-user/#join-an-organization).
```